### ISSUE TITLE
Screen reader doesn’t announce dynamic changes in External Connection URL field

### ISSUE THREAD

**Issue by samikeijonen** [10up/distributor#554]
**Is your enhancement related to a problem? Please describe.**
When setting External Connection URL in `post-new.php?post_type=dt_ext_connection`, screen reader doesn't announce dynamic changes happening below the field.

**Describe the solution you'd like**
Use `[wp.a11y.speak()](https://make.wordpress.org/accessibility/handbook/markup/wp-a11y-speak/)` to announce dynamic changes. Or [WP a11y package](https://developer.wordpress.org/block-editor/packages/packages-a11y/) for `speak`.

**Designs**
<img width="437" alt="Connection extra field" src="https://user-images.githubusercontent.com/1820415/76835159-2adf7900-6837-11ea-842a-19f2d9b89546.png">
